PaymentStatementIntegrations/FfcCrmInsert/workflow.json

```diff
@@ -1,809 +1,1009 @@
-                "value": "@appsetting('PortalClientSecret')",
-                "defaultValue": "@appsetting('PortalClientSecret')"
-            },
-            "SharepointClientId": {
-                "type": "String",
-                "value": "@appsetting('SharepointClientId')",
-                "defaultValue": "@appsetting('SharepointClientId')"
-            },
-            "SharepointTenantId": {
-                "type": "String",
-                "value": "@appsetting('SharepointTenantId')",
-                "defaultValue": "@appsetting('SharepointTenantId')"
-            },
-            "SharepointClientSecret": {
-                "type": "String",
-                "value": "@appsetting('SharepointClientSecret')",
-                "defaultValue": "@appsetting('SharepointClientSecret')"
-            },
-            "UnknownContactId": {
-                "type": "String",
-                "value": "@appsetting('UnknownContactId')",
-                "defaultValue": "@appsetting('UnknownContactId')"
-            },
-            "FfcRetrievalWorkflowSasToken": {
-                "type": "String",
-                "value": "@appsetting('FfcRetrievalWorkflowSasToken')",
-                "defaultValue": "@appsetting('FfcRetrievalWorkflowSasToken')"
-            },
-            "FfcRetrievalWorkflowEndpoint": {
-                "type": "String",
-                "value": "@appsetting('FfcRetrievalWorkflowEndpoint')",
-                "defaultValue": "@appsetting('FfcRetrievalWorkflowEndpoint')"
-            },
-            "FfcServiceBusTopic": {
-                "type": "String",
-                "value": "@appsetting('FfcServiceBusTopic')"
-            },
-            "SharepointSiteSubPath": {
-                "type": "String",
-                "value": "@appsetting('SharepointSiteSubPath')",
-                "defaultValue": "@appsetting('SharepointSiteSubPath')"
-            }
+        "runAfter": {
+          "Initialize_filename": [
+            "SUCCEEDED",
+            "TIMEDOUT",
+            "FAILED",
+            "SKIPPED"
+          ]
         },
-        "triggers": {
-            "When_messages_are_available_in_a_topic_subscription_(peek-lock)": {
-                "type": "ServiceProvider",
-                "inputs": {
-                    "parameters": {
-                        "topicName": "@appsetting('FfcServiceBusTopic')",
-                        "subscriptionName": "crm",
-                        "isSessionsEnabled": false
-                    },
-                    "serviceProviderConfiguration": {
-                        "connectionName": "serviceBus",
-                        "operationId": "peekLockTopicMessages",
-                        "serviceProviderId": "/serviceProviders/serviceBus"
-                    }
-                },
-                "splitOn": "@triggerOutputs()?['body']"
-            }
-        }
+        "trackedProperties": {},
+        "type": "Scope"
+      }
+    },
+    "contentVersion": "1.1.0.0",
```

Comment:
I would manually increment this version number so you can easily check if the correct version of the workflow has deployed ok (by inspecting the workflow JSON once deployed)

Reply:
Version updated to 1.2.0.0